src/source_map_support.js

```diff
@@ -0,0 +1,108 @@
+var vlqMap = {};
+'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/='.split('').forEach(function (c, i) {
+  vlqMap[c] = i;
+});
+
+function WASMSourceMap(sourceMap) {
```

Comment:
How about `WasmSourceMap` ("wasm" isn't an acronym, so it feels a little odd to capitalize all of it)?